### DISCUSSION TITLE
License references not clear

### DISCUSSION THREAD

**Discussion by rahulmohang** [eslint/eslint#17064] [General] [answered]
Hi, 

I have found two license references that are not very clear:

eslint-8.34.0/tests/lib/rules/prefer-object-spread.js
eslint-8.34.0/lib/rules/prefer-object-spread.js

The author refers to LICENSE file in the root directory. My assumption is that the author refers to 'eslint-8.34.0/LICENSE'. However, since there is a possibility that the two files mentioned above could be taken from another project that also might have LICENSE file in the root directory, I would like to confirm that the license referred to is the MIT license in the file, 'eslint-8.34.0/LICENSE'.

**Comment by mdjermanovic** [comment]
Hi @rahulmohang!

To confirm, do you mean these two lines?

https://github.com/eslint/eslint/blob/cf682d249f04a6a304407d5b9ddbbc4a9714dd62/lib/rules/prefer-object-spread.js#L4

https://github.com/eslint/eslint/blob/cf682d249f04a6a304407d5b9ddbbc4a9714dd62/tests/lib/rules/prefer-object-spread.js#L4

**Reply by rahulmohang** [reply]
Hi @mdjermanovic,

Yes.

**Comment by nzakas** [comment, marked as answer]
All of the files in the `eslint` repo originate from the ESLint project unless otherwise indicated in a comment inside of the file. So, unless there is a comment in the file explicitly stating a separate license and copyright, you can assume all files fall under the MIT license contained in the LICENSE file in the root of `eslint` repo.

**Reply by rahulmohang** [reply]
Hi @nzakas,

Thank you for the information.
